akka-http-core/src/main/scala/akka/http/impl/settings/ConnectionPoolSettingsImpl.scala

```diff
@@ -33,6 +36,7 @@ object ConnectionPoolSettingsImpl extends SettingsCompanion[ConnectionPoolSettin
   def fromSubConfig(root: Config, c: Config) = {
     ConnectionPoolSettingsImpl(
       c getInt "max-connections",
+      c getIntOpt "min-connections" getOrElse 0,
```

Comment:
Just to confirm - will it be a compatible change? (Assuming I will uncomment the line in reference.conf) ?